views/ctltools/forms/add-ctl-tool.hbs

```diff
@@ -27,6 +27,9 @@
     <input type="text" name="medium[mediumId]" id="mediumId" class="form-control" placeholder="urn:bilo:medium:123456789">
 </div>
 <div class="form-group">
+    <label class="control-label" for="mediaSourceId">Medienkatalog-ID</label>
```

Comment:
Why ID is in upper letter while "Medium-Id"?

Reply:
changed to Id, i also questioned the value of the placeholder and am waiting for response